test/EntityFramework6.Npgsql.Tests/EntityFrameworkMigrationTests.cs

```diff
@@ -524,6 +524,20 @@ public void MoveTableOperationNewSchemaIsNull()
         }
 
         [Test]
+        public void MoveTableOperationPrequotedSchema()
+        {
+            var operations = new List<MigrationOperation>();
+            operations.Add(new MoveTableOperation("someOldSchema.someTable", "\"prequotedNewSchema\""));
```

Comment:
Table names still just add quotes so using `"someOldSchema"."someTable"` will not work.
I guess table names should also use AppendQuotedIdentifier, but that's a separate issue.
Until then I test for `"someOldSchema".someTable`.